homeassistant/components/mobile_app/notify.py

```diff
@@ -89,13 +89,12 @@ def targets(self):
         targets = kwargs.get(ATTR_TARGET)
 
         if not targets:
```

Comment:
Isn't this more explicit?
```suggestion
        if targets is None:
```

Reply:
It tests two different things. The first will also test for empty lists.

Reply:
Ah, right. Didn't think about that.